### PR TITLE
Simplify most vehicle descriptions

### PR DESCRIPTION
**Pull request by Wuzzy2**
Here's a follow-up PR to the previous ride/vehicle description rework (#52).

This further refines the vehicle descriptions. This time, by simplifying them and moving the focus away from the track to the vehicle itself.
Many descriptions, especially for roller coaster trains, were still mostly a direct copy from the ride descriptions, and thus included many words about the track. As the vehicle descriptions appear in the vehicle tab of the ride window, descriptions of the track are redundant; the track is already described just fine in the Build Ride window.
The vehicle descriptions are changed in a way that they only describe features relevant to the vehicle, rather than the track. There are some exceptions, but even for those I tried to kept words about the track itself ot a minimum.

Other noteworthy changes:
* “dodgem cars” renamed to “dodgems”. I looked it up in dictionaries, and I think in UK the bumper cars are really just called “dodgems”, and not “dodgem cars”.
* Reworked descriptions for Dodgems and Flying Saucer and Go-Karts (and all their variants) to follow the same style. They are now written in a more generic way so that they make sense both as ride *and* vehicle description. <https://github.com/OpenRCT2/OpenRCT2/issues/10194> will be ALMOST fixed by this (still needs edit of `en-GB.txt` for the go-kart ride, expect a PR on OpenRCT2 repo soon)
* Various minor fixes, mostly grammar-related

### CODE REVIEW

File: objects/rct2tt/ride/rct2.tt.hoverbke.json [hunk 1]
```diff
@@ -60,7 +60,7 @@
             "ru-RU": "Ховербайки"
         },
         "description": {
-            "en-GB": "Riders sit on futuristic bike-shaped vehicles which travel along a single-rail track",
+            "en-GB": "Futuristic bike-shaped single vehicles",
```
Comment (Gymnasiast) [inline objects/rct2tt/ride/rct2.tt.hoverbke.json]: That doesn't sound right... don't you mean `single-rider vehicles`?

**Comment by Wuzzy2** [comment]
@Gymnasiast, @duncanspumpkin: Requesting review.

**Comment by Wuzzy2** [comment]
Bump.

**Comment by duncanspumpkin** [comment]
I don't quite see the point of the renames but then I know little about UI design I leave it to @Gymnasiast to decide.

File: objects/rct2tt/ride/rct2.tt.bmvoctps.json [hunk 2]
```diff
@@ -141,7 +141,7 @@
             "ru-RU": "Падение из космоса"
         },
         "description": {
-            "en-GB": "A themed rotating observation cabin which travels up a tall tower",
+            "en-GB": "A themed rotating observation cabin",
```
Comment (Fred-1044276) [inline objects/rct2tt/ride/rct2.tt.bmvoctps.json]: It should be something like this "A themed rotating observation cabin shaped like a blob from a sci-fi B-movie"

**Comment by Wuzzy2** [comment]
Thanks, Fred for looking at this. Note I will not do anything in this PR until I got a review from Gymnasiast. I'm collecting complaints like this one so I can fix them all in one go.

**Comment by Gymnasiast** [comment]
I will give feedback before the end of March.

File: objects/rct2/ride/rct2.ding1.json [hunk 3]
```diff
@@ -65,7 +65,7 @@
             "ru-RU": "Шлюпки"
         },
         "description": {
-            "en-GB": "Riders travel in inflatable dinghies down a twisting semi-circular or completely enclosed tube track",
+            "en-GB": "Inflatable dinghies that are twisting down a semi-circular or completely enclosed tube track",
```
Comment (Gymnasiast) [inline objects/rct2/ride/rct2.ding1.json]: This still seems to be written from the track's perspective.

File: objects/rct2/ride/rct2.kart1.json [hunk 4]
```diff
@@ -84,8 +84,8 @@
             "ru-RU": "Картинг"
         },
         "description": {
-            "en-GB": "Self-drive petrol-engined go karts",
-            "en-US": "Self-drive gas-engined go karts",
+            "en-GB": "Riders race each other in self-drive petrol-engined go-karts",
```
Comment (Gymnasiast) [inline objects/rct2/ride/rct2.kart1.json]: I prefer reverting the description change (apart from the spelling of "go-karts"). This description is only shown in the vehicle window, for which I think the old description is more appropriate.

File: objects/rct2/ride/rct2.swans.json [hunk 5]
```diff
@@ -60,7 +60,7 @@
             "ru-RU": "Лебеди"
         },
         "description": {
-            "en-GB": "Swan shaped boat, propelled by the pedalling front seat passengers",
+            "en-GB": "Swan-shaped boats, propelled by the pedalling front seat passengers",
             "en-US": "Swan shaped boat, propelled by the pedaling front seat passengers",
```
Comment (Gymnasiast) [inline objects/rct2/ride/rct2.swans.json]: Also needs adjusting of en-US.

File: objects/rct2/ride/rct2.vekst.json [hunk 6]
```diff
@@ -102,8 +102,8 @@
             "ru-RU": "Лежачие горки"
         },
         "description": {
-            "en-GB": "Riders are held in special harnesses in a lying-down position, travelling through twisted track and inversions either on their backs or facing the ground",
-            "en-US": "Riders are held in special harnesses in a lying-down position, traveling through twisted track and inversions either on their backs or facing the ground",
+            "en-GB": "Riders are held in special harnesses in a lying-down position, travelling either on their backs or facing the ground",
```
Comment (Gymnasiast) [inline objects/rct2/ride/rct2.vekst.json]: Still written from the track's perspective.

File: objects/rct2tt/ride/rct2.tt.cerberus.json [hunk 7]
```diff
@@ -105,7 +105,7 @@
             "ru-RU": "Горки Цербер"
         },
         "description": {
-            "en-GB": "Sitting in comfortable trains with only simple lap restraints riders enjoy giant smooth drops and twisting track as well as plenty of 'air time' over the hills",
+            "en-GB": "Spacious trains with simple lap restraints with cars shaped like a multi-headed dog from the Underworld",
```
Comment (Gymnasiast) [inline objects/rct2tt/ride/rct2.tt.cerberus.json]:
```suggestion
            "en-GB": "Spacious trains with simple lap restraints with cars shaped like the multi-headed dog from the Underworld",
```

File: objects/rct2tt/ride/rct2.tt.jousting.json [hunk 8]
```diff
@@ -56,7 +56,7 @@
             "ru-RU": "Рыцарские горки"
         },
         "description": {
-            "en-GB": "Riders sit on the back of horse-shaped vehicles along a single-rail track",
+            "en-GB": "Single horse-shaped vehicles",
```
Comment (Gymnasiast) [inline objects/rct2tt/ride/rct2.tt.jousting.json]: I think "separate" would fit better than "single".

File: objects/rct2/ride/rct2.ding1.json (hunk 3 above)
Comment (Fred-1044276) [reply objects/rct2/ride/rct2.ding1.json]: It should be "Inflatable dinghies that can twist down a semi-circular or completely enclosed tube track"

File: objects/rct2tt/ride/rct2.tt.cerberus.json (hunk 7 above)
Comment (Fred-1044276) [reply objects/rct2tt/ride/rct2.tt.cerberus.json]: Or if you want to be consistent with the Hyper-Twister Trains (wide cars) that the Cerberus Trains are based on, then: "Wide 4-across cars with raised seating and simple lap restraints, themed to look like the multi-headed dog from the Underworld"

File: objects/rct2ww/ride/rct2.ww.sputnikr.json [hunk 9]
```diff
@@ -58,7 +58,7 @@
             "ru-RU": "Спутник"
         },
         "description": {
-            "en-GB": "Passengers ride face-down in a lying position, suspended in a specially designed car from the single-rail track, swinging freely from side to side around corners",
+            "en-GB": "Suspended satellite-shaped cars in which the passengers ride face-down in a lying position, swinging freely from side to side",
```
Comment (Fred-1044276) [inline objects/rct2ww/ride/rct2.ww.sputnikr.json]: The Sputnik Cars behave more like the regular Swinging Cars stat-wise, so I would have the description of: "Russian satellite-shaped cars which swing from the rail above"

**Comment by Gymnasiast** [comment]
>  They are now written in a more generic way so that they make sense both as ride and vehicle description

There is no need for that. The description shown for the ride is stored in `en-GB.txt` (except for flat rides and shops).

**Comment by Wuzzy2** [comment]
> There is no need for that. The description shown for the ride is stored in en-GB.txt (except for flat rides and shops).

Agreed. I have written the PR when the game still used the same string for both vehicle and ride for *some* ride types (this PR is old). At at least that's what I believe (maybe I'm wrong). Anyway, now the reason for this generic style is clearly not correct (anymore?), so yeah, this change is now pointless. I need to adapt the strings accordingly.

**Comment by Wuzzy2** [comment]
>              "en-GB": "Riders are held in special harnesses in a lying-down position, travelling either on their backs or facing the ground", 

> Still written from the track's perspective.

I don't see how.

File: objects/rct2tt/ride/rct2.tt.1920racr.json [hunk 10]
```diff
@@ -84,7 +84,7 @@
             "ru-RU": "Машинки 1920гг"
         },
         "description": {
-            "en-GB": "Riders race each other in 1920's race car themed go-karts",
+            "en-GB": "1920's race car themed go-karts",
```
Comment (Gymnasiast) [inline objects/rct2tt/ride/rct2.tt.1920racr.json]:
```suggestion
            "en-GB": "1920s race car themed go-karts",
```

File: objects/rct2tt/ride/rct2.tt.rivrstyx.json [hunk 11]
```diff
@@ -58,7 +58,7 @@
             "ru-RU": "Стикс"
         },
         "description": {
-            "en-GB": "Boats gently meander around a river track",
+            "en-GB": "Boats built for flat river tracks with an Underworld theme",
```
Comment (Gymnasiast) [inline objects/rct2tt/ride/rct2.tt.rivrstyx.json:61]:
```suggestion
            "en-GB": "Boats with an Underworld theme, built for flat river tracks",
```

**Pull request merged**